### ISSUE TITLE
Entity more info dialog shows scrollbar

### ISSUE THREAD

**Issue by joostlek** [home-assistant/frontend#23520]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

![image](https://github.com/user-attachments/assets/70ee253e-f604-4c82-8b12-af1748faf0c6)
![image](https://github.com/user-attachments/assets/e2108f3d-8952-4ac1-9a0e-4d2c09900527)

### Describe the behavior you expected

It should not have that scrollbar

### Steps to reproduce the issue

1. Open the entity dialog
2. Profit

### What version of Home Assistant Core has the issue?

core-2025.1.0b3

**Comment by silamon** [comment]
I can't reproduce it, unfortunately.

**Comment by joostlek** [comment]
Hah, Bram also messages me about it and I can't either anymore

**Comment by joostlek** [comment]
![image](https://github.com/user-attachments/assets/88c98530-6cf3-4d34-8f31-ba07498f86e6)

1. Click a dialog
2. Profit

**Comment by joostlek** [comment]
Right, this is reproducible with 2025.1.0, it sometimes just takes a little while before it works

**Comment by ildar170975** [comment]
@joostlek 
Could it be caused by some cached js?
Do not see it in my setup:
![image](https://github.com/user-attachments/assets/1804670f-4b49-41e8-be19-565d954704df)

**Comment by bramkragten** [comment]
Seems like it is related to tooltips, can someone confirm? Hover over the last changed time.

**Comment by silamon** [comment]
> Seems like it is related to tooltips, can someone confirm? Hover over the last changed time.

It is already confirmed to be a duplicate of #23588 . Just forgot to post it.
